modules/nw-networkpolicy-audit-concept.adoc

```diff
@@ -38,16 +93,17 @@ The logging format is compatible with syslog as defined by RFC5424. The syslog f
 
 The following table describes namespace annotation values:
 
-.Audit logging namespace annotation
+.Audit logging namespace annotation for `k8s.ovn.org/acl-logging`
 [cols=".^4,.^6a",options="header"]
 |====
-|Annotation|Value
+|Field|Description
 
-|`k8s.ovn.org/acl-logging`
-|
-You must specify at least one of `allow`, `deny`, or both to enable audit logging for a namespace.
+|`deny`
+|Blocks namespace access to any traffic that matches an ACL rule with the `deny` action. The field supports `alert`, `warning`, `notice`, `info`, or `debug` values.
 
-`deny`:: Optional: Specify `alert`, `warning`, `notice`, `info`, or `debug`.
-`allow`:: Optional: Specify `alert`, `warning`, `notice`, `info`, or `debug`.
+|`allow`
+|Permits namespace access to any traffic that matches an ACL rule with the `allow` action. The field supports `alert`, `warning`, `notice`, `info`, or `debug` values.
 
+|`pass`
+|A `pass` action applies access control to an ACL rule. A network policy does not support a `pass` action.
```

Comment:
A 'pass' action applies to admin network policy ACL rule

Reply:
:heavy_check_mark: